Cargo.lock

```diff
@@ -5826,7 +5828,6 @@ dependencies = [
 [[package]]
 name = "sgx-runtime"
 version = "0.8.0"
-source = "git+https://github.com/ajuna-network/sgx-runtime?branch=integritee-network-master#b1968d0866f53d6387606bc4d3d306f80450a021"
```

Comment:
Is this cargo.lock correct? It seems strange that the source path is completely removed.